org.tug.texworks.yaml

```diff
@@ -0,0 +1,143 @@
+app-id: org.tug.texworks
+runtime: org.kde.Platform
+runtime-version: '5.13'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+command: texworks
+rename-desktop-file: texworks.desktop
+rename-appdata-file: texworks.appdata.xml
+rename-icon: TeXworks
+finish-args:
+#  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
+  - --env=TEXMFCACHE=$XDG_CACHE_HOME
+  - --env=SESSION_MANAGER= # required to avoid Qt warning
+  - --env=PATH=/app/texlive/bin/x86_64-linux:/app/bin:/usr/bin # required to find texlive binaries
+modules:
+  - name: texlive
+    buildsystem: simple
+    build-commands:
+      # Don't use /usr/lib/sdk/texlive/install.sh as that copies everthing into
+      # /app rather than a subdirectory, causing massive clutter of that folder
+      - install -d /app/texlive
+      - cp -ra /usr/lib/sdk/texlive/* /app/texlive
+
+  - name: openjpeg # required by poppler
```

Comment:
Could you use the openjpg module from the shared modules here?

Reply:
Done in 952cb5e